bin/plotting/pycbc_plot_singles_vs_params

```diff
@@ -123,6 +139,29 @@ if opts.max_y is not None:
 x = x[mask]
 y = y[mask]
 
+title = '%s of %s triggers over %s and %s' % (opts.z_var.title(),
+                         opts.detector, opts.x_var.title(), opts.y_var.title())
+fig_caption = ("This plot shows the %s of single detector triggers for the %s "
+               "detector. %s is shown on the colorbar axis against %s and %s "
+               "on the x- and y-axes." % (opts.z_var, opts.detector,
+                                   opts.z_var.title(), opts.x_var, opts.y_var))
+
+if not any(mask):
+    # All triggers removed - make a blank plot which says so:
+    fig = pl.figure()
+    ax = fig.gca()
+    pl.text(0.5, 0.5, 'no triggers in the range')
+
+    pycbc.results.save_fig_with_metadata(
+        fig,
+        opts.output_file,
+        title=title,
+        caption=fig_caption,
+        cmd=' '.join(sys.argv),
+        fig_kwds={'dpi': 200}
+    )
+    sys.exit()
+
```

Comment:
This can be removed if wanted - it is basically a safety for if there aren't many triggers - this is a consequence of applying the `--min-z`